src/develop/imageop.c

```diff
@@ -3077,7 +3077,10 @@ void dt_iop_cancel_history_update(dt_iop_module_t *module)
 
 char *dt_iop_warning_message(const char *message)
 {
-  return g_strdup_printf("<span foreground='red'>⚠</span> %s", message);
+  if(dt_conf_get_bool("plugins/darkroom/show_warnings"))
```

Comment:
This seems to only remove the "red" on the message. But it is still displayed or not ?

Reply:
This part acts on the module title, despite what the function name says (I got confused too). We simply remove the warning sign here, then the module title shows as usual.